src/components/reusable/table/table-expanded-row.ts

```diff
@@ -0,0 +1,50 @@
+import { LitElement, html } from 'lit';
+import { property, customElement } from 'lit/decorators.js';
+
+import styles from './table-expanded-row.scss';
+
+/**
+ *
+ * `kyn-expanded-tr` Web Component.
+
+ * Designed to display additional details for a row in a table.
+ * The row is expandable and can be expanded/collapsed by toggling the plus/minus icons.
+ *
+ * @slot unnamed - The slot for adding content to the expandable details section.
+ */
+@customElement('kyn-expanded-tr')
+export class TableExpandedRow extends LitElement {
+  static override styles = [styles];
+
+  /**
+   * The number of columns that the expanded row should span.
+   * Reflects the `colspan` attribute.
+   */
+  @property({ type: Number, attribute: 'colspan' })
+  colSpan = 1;
+
+  /**
+   * `true` if the table row should be expanded.
+   */
+  @property({ type: Boolean, reflect: true })
+  expanded = false;
+
+  override render() {
+    const { colSpan } = this;
+    return html`
+      <td></td>
```

Comment:
Let's just remove this empty `<td>`, have the colspan fill the entire row, and instead give some padding to the entire expanded row container. Right now the nested table blends into the next row of the parent table which is confusing.

Reply:
The parent table columns will never align with the child table columns because they are standalone, and need to be treated as such. I've mentioned this several times to make sure it was clear to the designers.

Reply:
Actually, they can be aligned. 
Anyways, I spoke with Robert, Marco, and Matthew, they're all ok with this empty td for now. Since it looks indented with the space of the expanding column which is nice enough. They'll look for more feedbacks and explore more things regarding table UX design and what should be the padding, (fixed, responisive...etc), and what values are good.
Also, apps have the full control over the rest of the space.

Reply:
Well, they can be aligned.
Anyways, I spoke with Matthew, Marco, and Robert about this indentation and this empty td. We'll all agree to leave it there. . UX is still gathering feedbacks and exploring things regarding table. A fixed padding-left of 64px is another option but it's fixed so not for now. Besides, changing this is not a breaking change by any means.

Reply:
Curious, how are they aligned? The only thing I can think of is fixed width columns.

Reply:
Yes, fixed widths.